### DISCUSSION TITLE
Déclaration handicap : ajout coches lors de la création du compte lycéen

### DISCUSSION THREAD

**Discussion by fabiend51** [unistra/esup-immersup-demandes#16] [7- Autres sujets]
lorsqu'un lycéen va créer son compte, nous souhaiterions rajouter 2 coches :
 1- une coche qui certifie sur l'honneur, en tant que majeur, qu'il a l'autorisation parentale pour s'inscrire et si cette coche n'est pas faite, le bouton "valider" n'est pas actif.
 2- une coche qui propose au lycéen d'être recontacté par la mission handicape (coche non bloquante)

Est-il possible de créer ces coches ?

merci

**Comment by efinkbeiner5** [comment]
Pour le cas 1, il faut utiliser les pièces justificatives et mettre un accord parental pour les mineurs. 
Pour le cas 2, on peut ajouter la coche dans la fiche Lycéen. Le problème est ensuite de savoir ce qu'on doit faire avec cette coche. On peut ajouter l'information dans une extraction pour que vous puissiez sortir une liste mais faut-il envoyer un mail et à qui au moment d'une inscription à un créneau précis ?

**Comment by efinkbeiner5** [comment]
Le sujet reste ouvert. Intéressant pour de nombreux établissement mais pas encore mature. Difficile de converger

**Comment by efinkbeiner5** [comment]
Réalisé en 2025
